admin/modules/config/thread_prefixes.php

```diff
@@ -530,7 +581,33 @@ function checkAction(id)
 		$table->construct_row();
 	}
 
-	$table->output($lang->thread_prefixes);
+	$table->output($title);
 
 	$page->output_footer();
 }
+
+function thread_prefix_sort($a, $b)
+{
+
+	if($a['forums'] === '-1' && $b['forums'] !== '-1')
+	{
+		return -1;
+	}
+
+	if($a['forums'] !== '-1' && $b['forums'] === '-1')
+	{
+		return 1;
+	}
+
+	if(count($a['forum_fids']) !== count($b['forum_fids'])) {
+		return count($b['forum_fids']) - count($a['forum_fids']);
+	}
+	elseif(count($a['forum_fids']) === 1)
+	{
+		$forums = cache_forums();
```

Comment:
Same here. It's worth noting that forum IDs might not exist due to data inconsistency, though rarely happens.